4001-5000/LIT4991Months.xml

```diff
@@ -45,12 +55,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         </profileDesc>
         <revisionDesc>
             <change who="DE" when="2018-06-29">Created entity</change>
+              <change who="DN" when="2022-10-11">Revision, upgrading to general record</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <div type="bibliography"/>
-         <!---->
+
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BNFabb134"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BNFet160#ms_i2.4"/>
+                    <relation name="saws:isDifferentTo" active="LIT4991Months" passive="LIT6251Months"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="IVorlov20"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLadd16211"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="EMIP00636"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BDLaeth34"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLadd24990"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BMLor148"/>
+                      <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLorient766"/>
+                      <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet40"/>
+                        <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet24"/>
+                        <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet15"/>
```

Comment:
what is with bnfabb39? this or another new one?

Reply:
(bnfabb39  has to be updated)

Reply:
Yes, added. Abb 39 will be updated.